### PR TITLE
added AMD64 CPU as a requirement

### PR DESCRIPTION
**Pull request by ayushpatil2122**
### Description
[kubernetes.io/docs/tasks/run-application/run-replicated-stateful-application/#statefulset
](https://kubernetes.io/docs/tasks/run-application/run-replicated-stateful-application/#statefulset)this page does not specify AMD64 CPU as a requirement

### CODE REVIEW

**Comment by thisisharrsh** [comment]
Hi @ayushpatil2122, Do we have any existing issues related to this fix?

**Comment by thisisharrsh** [comment]
_Page related to this fix:: https://kubernetes.io/docs/tasks/run-application/run-replicated-stateful-application/#before-you-begin_

**Comment by thisisharrsh** [comment]
_Deployment preview: https://deploy-preview-48253--kubernetes-io-main-staging.netlify.app/docs/tasks/run-application/run-replicated-stateful-application/#before-you-begin_

**Review by sftim — COMMENTED** [review]
/lgtm

(with a nit)

File: content/en/docs/tasks/run-application/run-replicated-stateful-application.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ on general patterns for running stateful applications in Kubernetes.
 - Some familiarity with MySQL helps, but this tutorial aims to present
   general patterns that should be useful for other systems.
 - You are using the default namespace or another namespace that does not contain any conflicting objects.
+- You need to have a AMD64 CPU.
```
[the remark this answers is not in the document]
Comment (thisisharrsh) [reply content/en/docs/tasks/run-application/run-replicated-stateful-application.md:40]: yes, As AMD64 would be very specific.

**Comment by ayushpatil2122** [comment]
@thisisharrsh  sorry for late reply but their is no existing issues related to this fix

**Pull request merged**